packages/dashboard/src/store/actions/bringToFront/index.ts

```diff
@@ -17,8 +18,16 @@ export const onBringWidgetsToFrontAction = (): BringWidgetsToFrontAction => ({
 export const bringWidgetsToFront = (state: DashboardState): DashboardState => {
   const widgets = state.dashboardConfiguration.widgets;
   const slectedWidgets = state.selectedWidgets;
+
+  const unselectedWidgets = xorBy(widgets, slectedWidgets, 'id');
```

Comment:
can't we just check that widgets length is equal to slectedWidgets length?

Reply:
ya, that would work too

Reply:
Oh actually we can't do that for this scenario. This list is also used to figure out the target z index for which selected widgets needs to be moved before / after. Previously we were just comparing against all widgets which would artificially inflate / defalte the zindex depending on the selection.

Reply:
Oh I see, my bad. Thanks for clarifying!

Reply:
Haha I was like I thought there was a reason I did this! I forgot until I looked at the code again.